packages/TorneloScoresheet/src/pages/GraphicalRecording/GraphicalRecording.tsx

```diff
@@ -65,8 +71,16 @@ const GraphicalRecording: React.FC = () => {
     }
   };
 
+  const handleSelectWinner = (player: Player) => {
+    setShowSignature(true);
+    setShowEndGame(false);
+    setSelectedWinner(player);
+  };
+
   const cancelSelection = () => {
+    setShowSignature(false);
     setShowEndGame(false);
+    setSelectedWinner(undefined);
   };
```

Comment:
Also I think these functions should go under the button params section, maybe write a comment on line 159 called 'Button functions' or something and move your button functions to join the others